application_sandbox/async_compute/main.cpp

```diff
@@ -980,16 +974,15 @@ class AsyncSample : public sample_application::Sample<AsyncFrameData> {
       VkBufferMemoryBarrier barrier = {
           VK_STRUCTURE_TYPE_BUFFER_MEMORY_BARRIER,  // sType
           nullptr,                                  // pNext
-          VK_ACCESS_SHADER_WRITE_BIT,               // srcAccessMask
```

Comment:
Were these changes just to clean up the existing async_compute sample?

Reply:
Yes, this was something I ran into and fixed in the new version, and backported to the original async_compute application. I split these out into separate commits to try to clarify this.